docs/en_US/NAS/HardwareAwareNAS.rst

```diff
@@ -3,27 +3,28 @@ Hardware-aware NAS
 
 .. contents::
 
-EndToEnd Multi-trial SPOS Demo
-------------------------------
+End-to-end Multi-trial SPOS Demo
+--------------------------------
 
 To empower affordable DNN on the edge and mobile devices, hardware-aware NAS searches both high accuracy and low latency models. In particular, the search algorithm only considers the models within the target latency constraints during the search process.
 
-To run this demo, first install nn-Meter from source code (Github repo link: https://github.com/microsoft/nn-Meter. Currently we haven't released this package, so development installation is required).
+To run this demo, first install nn-Meter by running:
 
 .. code-block:: bash
 
-  python setup.py develop
+  pip install nn-meter
 
 Then run multi-trail SPOS demo:
 
 .. code-block:: bash
 
   python ${NNI_ROOT}/examples/nas/oneshot/spos/multi_trial.py
 
+
 How the demo works
 ------------------
```

Comment:
Since there are two examples, this should better be second level title.

Reply:
agree